images/vm-image-driver.bb

```diff
@@ -7,6 +7,9 @@ inherit image-hostname vm-image-features vm-guest-images-install
 
 SUMMARY = "Driver VM"
 
+IMAGE_INSTALL:append = " ${@bb.utils.contains("MACHINE_FEATURES", "pci", "pciutils", "", d)} "
+IMAGE_INSTALL:append = " ${@bb.utils.contains("MACHINE_FEATURES", "usbhost", "usbutils", "", d)} "
```

Comment:
I suggest to use single append here, so it is easier and less verbose to add more packages to the image:
```
IMAGE_INSTALL:append = " \
    ${@bb.utils.contains("MACHINE_FEATURES", "pci", "pciutils", "", d)} \
    ${@bb.utils.contains("MACHINE_FEATURES", "usbhost", "usbutils", "", d)} \
    "
```

In addition this change could be split to separate commit from kernel patch. Splitting is just my recommendation - I'll leave it to your consideration :)